eng/scripts/patch-vs.ps1

```diff
@@ -0,0 +1,64 @@
+Param (
```

Comment:
The only thing I think that's missing is a comment either at the top of this file or in the .yaml that says this is a hack until the next Dev17 public preview.